akka-stream/src/main/scala/akka/stream/serialization/StreamRefSerializer.scala

```diff
@@ -86,7 +87,7 @@ private[akka] final class StreamRefSerializer(val system: ExtendedActorSystem)
 
   private def serializeRemoteSinkFailure(
       d: StreamRefsProtocol.RemoteStreamFailure): StreamRefMessages.RemoteStreamFailure = {
-    StreamRefMessages.RemoteStreamFailure.newBuilder().setCause(ByteString.copyFrom(d.msg.getBytes)).build()
+    StreamRefMessages.RemoteStreamFailure.newBuilder().setCause(UnsafeByteOperations.unsafeWrap(d.msg.getBytes)).build()
```

Comment:
`String.getBytes` is not safe, as it uses the platform charset: https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html#getBytes()

Reply:
good catch, there is actually another usage that we can change at the same time

https://github.com/akka/akka/blob/622d8af0ef9f685ee1e91b04177926ca938376ac/akka-actor/src/main/scala-2.12/akka/util/ByteString.scala#L157

It's only used from tests and not public, but anyway.

Reply:
Since it already is like that, it will potentially be wire incompatible to change it, but otoh it is not on any hot path and should rarely happen (stream failing in the middle of rolling upgrade of Akka using streamrefs) so I think it is fine.

Reply:
Ah, the receiving side already expected UTF-8, so no wire incompatibility. Only fixing something potentially broken. 👍